alchemy/package.json

```diff
@@ -23,6 +23,7 @@
     "./github": "./lib/github/index.js",
     "./neon": "./lib/neon/index.js",
     "./os": "./lib/os/index.js",
+    "./serde": "./lib/serde/index.js",
```

Comment:
I think we can just add them to the root index

```ts
// alchemy/src/index.ts
export * from "./serde.js";
```

Reply:
Cool, I'll do that. I'll also update docs to match!

Reply:
Resolved via 0c123accf43ef83ae57c6de4b7b44b5334fa855f